### PR TITLE
feat(meetings-sample): add devices section

### PR DESCRIPTION
**Pull request by InteractiveTimmy**
# Pull Request

## Description

The scope of the changes in this pull request are focused around adding the meetings devices section from the original `plugin-meetings-sample` to `meetings-sample`.

Fixes # [SPARK-143731](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-143731)

### CODE REVIEW

File: packages/node_modules/samples/browser-meetings/app.js [hunk 1]
```diff
@@ -372,3 +372,201 @@ function sendDtmfTones() {
...
+
+function claimPersonalMeetingRoom() {
+  console.log('DevicesControls#claimPersonalMeetingRoom()');
+
+  pmrDetailsElm.innerHTML = 'Attempting to claim';
+
+  const pmrId = pmrIdElm.value;
+  const pmrPin = pmrPinElm.value;
+
+  if (!pmrId && !pmrPin) {
```
Comment (abailey-wwt) [inline packages/node_modules/samples/browser-meetings/app.js:553]: Looking at the error thrown below, did you mean `!pmrId || !pmrPin`? It seems as if you are requiring both to be present, unless I misunderstood.

**Comment by abailey-wwt** [comment]
Unfortunately I don't have a device to test but the code looks good. I just had one minor comment.

File: packages/node_modules/samples/browser-meetings/index.html [hunk 2]
```diff
@@ -279,7 +285,28 @@ <h2>Meetings Devices</h2>
...
+          <br />
+          <button id="device-request-pin" type="button" onclick="webex.devicemanager.requestPin(currentDevice)">webex.devicemanager.requestPin()</button>
+          <input id="device-pin-query" name="devicePinQuery" placeholder="PIN" value="" type="number" />
+          <button id="device-request-pair" type="button" onclick="deviceRequestPair()">webex.devicemanager.pair()</button>
+          <button id="device-request-unpair" type="button" onclick="deviceRequestUnpair()">webex.devicemanager.unpair()</button>
+          <br />
+          <label id="device-pair-state">Not paired</label>
+          <br />
+          <br />
+          <label>Device Audio State</label>
```
Comment (mwegman) [inline packages/node_modules/samples/browser-meetings/index.html:300]: We could mention here that the state isn't 'live' and will need to be updated by hitting the 'deviceGetAudioState' button

File: packages/node_modules/samples/browser-meetings/app.js (hunk 1 above)
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/app.js:553]: Good catch! updating this now.

File: packages/node_modules/samples/browser-meetings/index.html (hunk 2 above)
Comment (InteractiveTimmy) [reply packages/node_modules/samples/browser-meetings/index.html:300]: Good idea, I'll proceed with making this change now.

**Comment by InteractiveTimmy** [comment]
All comments have been addressed as of now.

**Pull request merged**